src/number_theory/bernoulli_polynomials.lean

```diff
@@ -90,6 +91,28 @@ end
 
 end examples
 
+lemma derivative_bernoulli_add_one (k : ℕ) :
+  (bernoulli (k + 1)).derivative = (k + 1) * bernoulli k :=
```

Comment:
Seems more consistent with other spellings in this file as:
```suggestion
  (bernoulli (k + 1)).derivative = (k + 1) • bernoulli k :=
```
(it's also a more useful form, probably)

Reply:
Actually this file is super-inconsistent anyway, so it doesn't matter.